### ISSUE TITLE
Possible Meeting Regarding Solving Techniques

### ISSUE THREAD

**Issue by jaamestaay** [jaamestaay/M2R-Group-29#9]
Dear Dr Olver, Daniel and Tianyi,

@LaiquendiJ and I have been going back and forth on the adjoint method since our meeting this morning. Having looked into the adjoint method a bit more, I'm starting to get quite confused about it myself. Here's what I have so far:

The literature seems to suggest that there are two different adjoint methods - I'll call them the discrete and continuous adjoint methods, which are methods used to calculate the gradients of the loss function $L$ with respect to $\theta$ that feed into the gradient descent algorithm. However, I'm not really sure what the differences between the two are, given that they (1) stem from the same equation and (2) seem to both require some level of discretisation (either discretisation at the function definition level or the discretisation at the evaluation stage) of the adjoint ODE to obtain the result. However, the literature isn't very clear on what their differences are exactly and how we can understand them.

I understand that this is outside of our current meeting schedule, but I'd like to ask if we could meet separately from the rest of the group to try to understand these differences. @LaiquendiJ and I are planning to meet at 2pm tomorrow - we can meet perhaps around that time as well. Thank you!

**Comment by dlfivefifty** [comment]
Sure

**Comment by jaamestaay** [comment]
Thanks Dr Olver! @LaiquendiJ and I will discuss the issue first before we come to meet you, say at 2:30pm. Speak to you then!

**Comment by jaamestaay** [comment]
Dear Dr Olver,

Thanks for meeting with us this afternoon - it's really helped us with the conceptual understanding. We've updated the contents page to reflect the ideas that we've discussed today:
- Discussing Gradient Descent and Loss Functions (essentially the optimisation problem of the whole neural network) in the Neural Network section
- Focussing the discussion of the Neural ODEs on the backpropagation of gradients through the Neural ODE layer (as part of the full backpropagation method for the Neural Network).

These changes can be found in the latest update of the README.md file, for your perusal.
